cax/main.py

```diff
@@ -288,10 +292,48 @@ def massive():
         docs = list(collection.find(query,
                                     sort=sort_key,
                                     projection=['start', 'number','name',
-                                                'detector', '_id']))
+                                                'detector', '_id', 'processor']))
 
         for doc in docs:
+            
+            #If no_batch_mode is chosen, no jobs will submitted to batch queues
+            if no_batch_mode == True:
+                
+                #Define basic command:
+                basic_command = """#!/bin/bash
+cax --once {config} --name {name}
```

Comment:
should be ok for a string definition, similar to [this](https://github.com/XENON1T/cax/blob/v5.2.1/cax/config.py#L241)